src/cmd/ksh93/tests/path.sh

```diff
@@ -247,7 +247,7 @@ y=$(whence rm)
 whence getconf > /dev/null  &&  err_exit 'getconf should not be found'
 #builtin /bin/getconf
 PATH=/bin
-PATH=$(getconf PATH)
+PATH=$(/usr/bin/getconf PATH)
```

Comment:
I've got a slightly different fix for this as part of my change for issue #158 that doesn't assume `getconf` is not in a particular directory and adds a TODO to remind us to fix this if and when getconf becomes a builtin again.